why3/src/coma.rs

```diff
@@ -235,16 +235,6 @@ impl Print for Expr {
     }
 }
 
-fn braces<'a, A: pretty::DocAllocator<'a>>(
-    doc: pretty::DocBuilder<'a, A>,
-) -> pretty::DocBuilder<'a, A> {
-    if !matches!(&*doc.1, pretty::Doc::Nil) {
-        doc.braces()
-    } else {
-        doc
-    }
-}
-
```

Comment:
This has nothing to do with this PR.

Reply:
Indeed but it got rid of a warning so I left it in.